pkg/clusteragent/orchestrator/status.go

```diff
@@ -103,3 +100,29 @@ func GetStatus(apiCl kubernetes.Interface) map[string]interface{} {
 
 	return status
 }
+
+func setClusterName(status map[string]interface{}) {
+	message := "No cluster name was detected. This means resource collection will not work."
+
+	hostname, err := util.GetHostname()
+	if err != nil {
+		status["ClusterNameError"] = fmt.Sprintf("Error detecting cluster name: %s.\n%s", err.Error(), message)
+	} else {
+		cName := clustername.GetClusterName(hostname)
```

Comment:
the following if statement can be directly inlined here `if cName := ... ; cName == ...`